### PR TITLE
Add dev map

### PR DESCRIPTION
**Pull request by ElectroJr**
Requires space-wizards/RobustToolbox/pull/3847

This PR adds a new map and sets this as the default map when using the development config preset. 

This is partially because the server will start up faster and because it will generally make testing stuff easier. E.g., its quite tedious to have wait for the server to start, open the spawn menu, and then spawn an urist & some food via the spawn menu to test force feeding & stripping DoAfters.

The behaviour can be disabled by just setting the map cvar to an empty string via command line args or the toml. Arguably, this should be the other way around so maybe this needs some discussion.

Also, I'm not really a mapper and the debug map isn't all that fleshed out and some stuff just isn't mapped yet. E.g., there is no proper atmos loop which could arguably make some testing harder. I just added what I thought would be useful, with more useful things closer to the spawn point. Some of the entities are just there for easy copying. The map was based on the station station salvage for a smidge of sovl.

https://user-images.githubusercontent.com/60421075/224576335-8fa06697-50d4-4850-a4d6-9e1e9150a553.mp4

### CODE REVIEW

File: Content.Shared/Entry/EntryPoint.cs [hunk 1]
```diff
@@ -48,6 +50,9 @@ public override void PostInit()
             // fake lag rand leads to messages arriving out of order. Sadly, networking is not robust enough, so for now
             // just leaving this disabled.
             // configMan.OverrideDefault(CVars.NetFakeLagRand, 0.01f);
+
+            if (IoCManager.Resolve<INetManager>().IsServer)
+                configMan.OverrideDefault(CCVars.GameMap, "Dev");
```
Comment (PJB3005) [inline Content.Shared/Entry/EntryPoint.cs:55]: This should go into the `development` config preset, not be code like this.

**Review by PJB3005 — DISMISSED** [review]

File: Content.Shared/Entry/EntryPoint.cs (hunk 1 above)
Comment (ElectroJr) [reply Content.Shared/Entry/EntryPoint.cs:55]: ah right, I forgot those exist now

**Comment by mirrorcult** [comment]
blessed glad someone got around to this

**Review by mirrorcult — APPROVED** [review]

**Comment by CaptainSqrBeard** [comment]
maybe replace holofans with air fans?

**Comment by moonheart08** [comment]
Test fail is not spurious, please make sure this has docks for evac/cargo.

**Pull request merged**